pom.xml

```
@@ -8,23 +8,22 @@
     <relativePath />
   </parent>
   <artifactId>gitlab-plugin</artifactId>
-  <version>${revision}${changelist}</version>
+  <version>1.7.1</version>
   <name>GitLab Plugin</name>
   <url>https://github.com/jenkinsci/${project.artifactId}</url>
   <packaging>hpi</packaging>
 
   <properties>
-    <revision>1.7.2</revision>
+    <revision>1.7.1</revision>
     <changelist>-SNAPSHOT</changelist>
-    <jenkins.version>2.361.4</jenkins.version>
+    <jenkins.version>2.375.2</jenkins.version>
```

The ["How to choose a base Jenkins version"](https://www.jenkins.io/doc/developer/plugin-development/choosing-jenkins-baseline/) page recommends either 2.361.4 or 2.346.3 unless there are exceptional circumstances that recommend a newer version.  As far as I can tell, we should remain with 2.361.4 for now as our base Jenkins version.
```suggestion
    <jenkins.version>2.361.4</jenkins.version>
```